web/src/app/alerts/CreateAlertDialog/DialogNavigation.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import { DialogActions, Button } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
+
+const useStyles = makeStyles(theme => ({
+  button: {
+    marginRight: theme.spacing(1),
+  },
+  dialogActions: {
+    display: 'flex',
+    flexDirection: 'row-reverse',
+    justifyContent: 'flex-start',
+  },
+}))
+
+const nextIsDisabled = (activeStep, formFields) => {
+  switch (activeStep) {
+    case 0:
+      return !(formFields.summary && formFields.details)
+    case 1:
+      return formFields.selectedServices.length === 0
+    default:
+      return false
+  }
+}
+
+export default props => {
+  const {
+    activeStep,
+    formFields,
+    onClose,
+    onLastStep,
+    onSubmit,
+    setActiveStep,
+    steps,
+  } = props
+  const classes = useStyles()
+
+  const stepForward = () => {
+    setActiveStep(prevActiveStep => prevActiveStep + 1)
+  }
+
+  const stepBackward = () => {
+    setActiveStep(prevActiveStep => prevActiveStep - 1)
+  }
+
+  const getNextBtnLabel = () => {
+    switch (activeStep) {
+      case steps.length - 1:
+        return 'Done'
+      case steps.length - 2:
+        return 'Submit'
+      default:
+        return 'Next'
+    }
+  }
+
+  const getBackBtnLabel = () => {
+    switch (activeStep) {
+      case 0:
+        return 'Cancel'
+      default:
+        return 'Back'
+    }
+  }
```

Comment:
this can be an if statement or ternary operation since it just has the 1 case:

```
const getBackBtnLabel = () => return activeStep === 0 ? 'Cancel' : 'Back'
```

Reply:
That's valid. I'm mostly bringing it up because my IDE was yelling at me :P